### ISSUE TITLE
Footer Styles

### ISSUE THREAD

**Issue by ryuhhnn** [agauniyal/wireframe#17]
Make the footer area look a little more clean and consider adding in a few different columns.

**Comment by frankinedinburgh** [comment]
I can try do a bit of work on this.

**Comment by Pakistanka** [comment]
Hello. I can try to help if it needs

**Comment by ryuhhnn** [comment]
@Pakistanka That would be awesome! Submit a PR when you're ready :)

**Comment by BilyanHadzhi** [comment]
Hello, @ryuhhnn! Are you accepting contributions? I've noticed you don't include a `meta` [viewport tag](https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag) in `base.jade`, leading to non-responsive pages.

I see great potential to contribute to this project. I don't know node, express, jade too well but I guess they should be easy to pick up.

**Comment by agauniyal** [comment]
@BilyanHadzhi Sure. Feel free to open Pull Requests against `develop` branch.

**Comment by ollyjohn** [comment]
Is this still needing help?  I'd be more than happy to lend a hand.

**Comment by agauniyal** [comment]
@ollyjohn Yep, till the issues aren't closed or a corresponding PR isn't open, feel free to work on them.

**Comment by BilyanHadzhi** [comment]
Is there another place for discussing the project, so we don't populate just this issue?

Anyway, I skimmed over the code and have a few suggestions:
* Adding a `meta` viewport tag in `base.jade` is a must
* Prevening the submit button from submitting the form:
  * It could be left as is, and expect the user to add `return false` to the form tag
  * We could have a class like `nosubmit`/`no-submit`/`form-nosubmit` which (via js) changes the `onsubmit` property of  any elements that has it to `return false`
* For the footer itself, we could have CSS classes such as `big`, `medium` and `small` which would be added to the `footer` element and based on them we could control how big the 'footer area' is.
  * For example, for `footer.big` we'd have a bigger area, like [this](http://www.bypeople.com/wp-content/uploads/2016/03/big-size-footer.jpg), which seems to be how big most footers are.

I'm willing to open issues for the things suggested (and work on this one as well).
I'm writing this so as to see if you agree with my suggestions about the footer, so I could implement them that way.

**Comment by ryuhhnn** [comment]
@BilyanHadzhi there isn't any place for discussion on this project currently, unless @agauniyal has implemented any? Outside of that, you're more than welcome to submit individual issues on these matters and keep related discussion within those!

**Comment by BilyanHadzhi** [comment]
I'm having trouble getting my SCSS to take effect. I created a new file is `/sass/` and imported it to `main.scss`, gulp doesn't seem to be noticing. Any help?

**Comment by ryuhhnn** [comment]
@BilyanHadzhi  Could you maybe fork over the repository and provide some steps to recreate?

**Comment by BilyanHadzhi** [comment]
@ryuhhnn Sure. What I did was:
1. fork and clone the repo
2. `git checkout develop`
3. `git branch footer-work && git checkout footer-work`
4. `npm install`
5. `touch sass/_footer.scss`
6. Write styles for a class in `_footer.scss` and `@import 'footer'` in `main.scss`
7. Run `gulp`
8. Add the class to an element in `base.jade` - element gets the class but no styles take effect

`gulp` also does not really do anything when I save any `.scss` file.

I could open a pull request with the code I have so we can see what I'm doing wrong?

**Comment by ryuhhnn** [comment]
@BilyanHadzhi could you please push changes to your forked repo? It would be better for me to take a look at it that way.

**Comment by BilyanHadzhi** [comment]
@ryuhhnn Just [did](https://github.com/BilyanHadzhi/wireframe/tree/footer-work). Also I see `public/css/main.css` was deleted after doing some gulp task.

**Comment by agauniyal** [comment]
@BilyanHadzhi @ryuhhnn https://gitter.im/discuss-wireframe/

**Comment by agauniyal** [comment]
@BilyanHadzhi gulp tasks might be in a mess atm, I haven't even seen `develop` branch for a while since I am busy with finishing my course. When I started this project I wanted to keep it minimal, that's why I picked only `sass` as dependency and kept out other things. Along the way some nice contributors added very good features which enhanced the project and added some extra things, which in my opinion aren't necessary. For example, we aren't really using server for doing anything except provide a short `localhost:port` url, which is already provided by browsersync gulp plugin.

So I'm thinking of working on this project again and restructure it well while keeping all essential things like styles and stuff and removing extraneous gulp plugins and server. I am also thinking of shifting to `component` based framework where user could do something like -

```
<html>
<head> ...</head>
<body>
<hero-image></hero-image>
<wide-slider></wide-slider>
<about-us>
    <p></p>
    <portfolio></portfolio>
</about-us>
<wide-footer></wide-footer>

```

which will be very nice for end user.

**Comment by BilyanHadzhi** [comment]
Yeah, I figured a rewrite may be the most appropriate thing to do. There are many deprecated node packages being used too since it has not been maintained for some time.
Overall, developing in its current state seems impossible to me.

**Comment by erickrod54** [comment]
Hello @agauniyal , is still open the issue ? i would like to work to do it :), can you assign it to me ?

**Comment by Chetax** [comment]
Hello @agauniyal , is still open the issue ?

**Comment by tinelisec** [comment]
Hello, if the issue is still open, i'd like to fix this issue, can you please assign me

**Comment by khushikushwah** [comment]
hi, if this issue is still open, I would like to work on this, can you please assign me
